### PR TITLE
[QE] Added tier1 Test cases

### PR DESCRIPTION
**Pull request by kpunwatk**
new file:   pkg/qe-tests/cypress/integration/tests/tier1/config_tier1.ts
	new file:   pkg/qe-tests/cypress/integration/tests/tier1/tier1_tests_vmware.ts
	new file:   pkg/qe-tests/cypress/integration/tests/tier1/tier1_vmware_tests_cleanup.ts

### CODE REVIEW

File: pkg/qe-tests/cypress/integration/tests/tier1/config_tier1.ts [hunk 1]
```diff
@@ -0,0 +1,370 @@
+import {
```
Comment (sshveta) [inline pkg/qe-tests/cypress/integration/tests/tier1/config_tier1.ts:1]: This is not a test file so I would suggest it should go under test folder .
According to this cypress framework guidelines .. It should go under integration/types/tier1/config_tier1.ts

**Review by sshveta — CHANGES_REQUESTED** [review]

File: pkg/qe-tests/cypress/integration/tests/tier1/config_tier1.ts (hunk 1 above)
Comment (sshveta) [reply pkg/qe-tests/cypress/integration/tests/tier1/config_tier1.ts:1]: Cypress framework guidelines -  https://docs.google.com/document/d/1IfApGDf1JmFHCf48XH3xk2noq6XJVBHv_tRuqMTqDBs/edit

Comment (kpunwatk) [reply pkg/qe-tests/cypress/integration/tests/tier1/config_tier1.ts:1]: Thanks @sshveta for the review! This is the config file for the tier1 test cases which contains the different provider details, mappings to be used for the tests and all the plans essentials requirements for running tests.
In types we have exported the data types required for the configs.
And we have always declared the required config files in the Test folder of particular providers and tiers.

Comment (sshveta) [reply pkg/qe-tests/cypress/integration/tests/tier1/config_tier1.ts:1]: @kpunwatk Okay if that's the framework guidelines your team follows . Please check with @ibragins too .
In MTA we keep only test files under test folder .

**Comment by yaacov** [comment]
closing as new development is moving to:
https://github.com/kubev2v/forklift-console-plugin

**Pull request closed without merging**